frontend/src/components/MainView/MainView.tsx

```diff
@@ -1,19 +1,73 @@
-import React from "react";
+import React, { useState } from "react";
 import styles from "./MainView.module.css";
 import Header from "../Header";
-import { Switch, Route, Redirect} from "react-router-dom";
+import { Switch, Route, Redirect } from "react-router-dom";
 import PrivateRoute from "../PrivateRoute";
 import HomePage from "../HomePage";
 import LogIn from "../LogIn";
 import RegistrationView from "../Registration";
 import ResetPassword from "../ResetPassword";
 import { getUserFromLocalStorage } from "../../app/utils";
+import { UserType } from '../../models/User.model'
+
+interface LoggedInViewProps {
+  onLogout?: Function
+}
+
+interface LoggedOutViewProps {
+  onLogin?: Function
+}
 
 const MainView: React.FC = () => {
-  
+  const [isLogged, setIsLogged] = useState(false);
+
+  const userData = getUserFromLocalStorage();
```

Comment:
After refresh, I see login form instead of home page.
To solve this problem use following code:
``` js
const userData = getUserFromLocalStorage();
const [isLogged, setIsLogged] = useState(Boolean(userData.userType));
```

Reply:
thanks, I've missed this pretty obvious scenario 😄 
fixed as you suggested